<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatShakerDataWidget</class>
 <widget class="QWidget" name="HeatShakerDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="upWidget" native="true">
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>振荡吸磁</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMagnetic">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>吸磁</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReset">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>模块复位</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHighSpeedShake">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>高速振荡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLowSpeedShake">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>低速振荡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStopShake">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>停止振荡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMagnetic_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>吸磁电机复位</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetParams">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>获取参数</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电机速度：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>K：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>B：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetParams_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>230</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>158</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>水平吸磁位：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMagnetic_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>吸磁电机运动</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>162</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>水平吸磁位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>返回位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>下拉吸磁位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上拉吸磁位</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>下拉吸磁位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>上拉吸磁位：</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="downWidget" native="true">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>加热</string>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>目标温度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
     </widget>
     <widget class="SwitchButton" name="swControlDuct" native="true">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>当前温度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHighSpeedShake_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>读取温度</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>老化次数：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="editTargetTemperature">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>当前轮次：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHighSpeedShake_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>190</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始老化</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHighSpeedShake_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>停止老化</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">hcscitoolkits/switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
